tests/components/zha/test_device_trigger.py

```diff
@@ -128,7 +146,15 @@ async def test_no_triggers(hass, mock_devices):
     reg_device = ha_device_registry.async_get_device({("zha", ieee_address)}, set())
 
     triggers = await async_get_device_automations(hass, "trigger", reg_device.id)
-    assert triggers == []
+    assert triggers == [
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)